tests/management/commands/create_fixtures.py

```diff
@@ -0,0 +1,49 @@
+"""Create serialized data for tests and store this data in a json file."""
+import os
+from contextlib import contextmanager
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+from pages.models import CustomPage
+from tests.catalog import models as catalog_models
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        self.prepare_db()
+        with self.save_to('search.json'):
+            self.create_custom_pages()
+            self.create_for_search()
+
+    def prepare_db(self):
+        call_command('migrate')
+
+    @contextmanager
+    def save_to(self, name: str):
```

Comment:
doesn't use self, and semantically it's separated function. It'll be good to move it to some `common.py` or smth.
Decide for your own plz